instrumentation/kamon-finagle/src/main/scala/kamon/instrumentation/finagle/client/Tags.scala

```diff
@@ -0,0 +1,63 @@
+package kamon.instrumentation.finagle.client
+
+import com.twitter.finagle.http.Status
+import com.twitter.util.Time
+import kamon.trace.Span
+
+import java.net.InetSocketAddress
+
+/**
+ * Unique tags and value constants added to Kamon Finagle spans.
+ *
+ * As a general rule metric tags should only be for tags with low cardinality.
+ */
+private[finagle] object Tags {
+
+  object Keys {
+    val ResourceName = "resource.name"
+    val SpanKind = "span.kind"
+    val SpanType = "span.type"
+    val ErrorType = "error.type"
+    val PeerPort = "peer.port"
+    val PeerHostIPV4 = "peer.ipv4"
+    val PeerHostname = "peer.hostname"
+    val HttpStatusCategory = "http.status_category"
+  }
+
+  object Values {
+    val ClientSpanKind = "client"
+    val HttpSpanType = "http"
+  }
+
+  import Keys._
+  import Values._
+
+  def setHttpClientTags(span: Span): Unit = {
+    span.tag(SpanKind, ClientSpanKind)
+    span.tag(SpanType, HttpSpanType)
```

Comment:
It shouldn't be necessary to add the span kind as a tag. The kind has its own field on the spans and it is set automatically by the HTTP Client instrumentation, and then the reporters turn it into a tag when the backend needs it. Here I don't think it is necessary.

Reply:
Got it. Yes, I see `span.kind` being added to tags.

The span type is an interesting case. We use the `kamon-datadog` span reporter and I noticed that the `DDSpan` that's serialized and sent to DataDog hardcodes its span kind field to `custom`. [There are a handful of supported options](https://docs.datadoghq.com/tracing/visualization/services_list/#services-types). They have subtle effects on how the APM frontends are rendered. We've since overridden this by implementing our own `KamonDataDogTranslator` to send it as `web`. I was thinking about opening a Kamon issue for this to make it more configurable, but our solution seems to work fine for now.

I'll remove this tagging from the implementation. I can sort out what our infrastructure needs outside of this instrumentation.

Reply:
I think that this is something we definitely should address in the datadog reporter itself. It wouldn't be too complicated to check if a span has `SpanKind=Client|Server` and some `http.*` tag, we would automatically add the span.type tag there so it works nicely in Datadog's UI. 

Also, I remember an old conversation about showing DB spans properly in Datadog. It might be a similar thing we need to do. Let's track those changes in a separate issue/pr: #1098